tests/test_executor.py

```diff
@@ -157,7 +157,7 @@ def nested_model(cond):
 
     @pm.model
     def outer_model():
-        cond = yield pm.HalfNormal("cond", 1)
+        cond = yield pm.HalfNormal("cond", 1, conditionally_independent=True)
```

Comment:
what happens without this condition?

Reply:
If this wasn't true, then the `sample_shape` that gets passed to the executor is ignored. I wanted to test how the executor works when it uses the supplied `sample_shape` (when `conditionally_independent=True`) and when it is ignored (all other distributions).